docs/changelog/2284.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix for creating virtualenvs from a Python 2.7 framework on macOS 12.3 - by :user:`nickhutchinson`.
```

Comment:
I consider this a feature rather than a bugfix, so let's move it to a feature file 👍 something like support for macos 12.3 👍